internal/common/schemafunc/json.go

```diff
@@ -1,42 +1,11 @@
 package schemafunc
 
 import (
-	"context"
 	"encoding/json"
 	"log"
 	"reflect"
-
-	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
 )
 
-func DiffSuppressJSON() planmodifier.String {
```

Comment:
This didn't work as expected. I was trying to add whitespace to the `pipeline` attribute, but still, an update was shown.
Worked when I instead used a `func (r *streamProcessorRS) ModifyPlan(ctx context.Context, req resource.ModifyPlanRequest, resp *resource.ModifyPlanResponse)` on the resource.
However, it is too hacky to include. Considering users adding additional whitespace to be a low chance.

Reply:
Not necessary. See above: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2519/files#r1728618273